draft-ietf-webtrans-overview.md

```diff
@@ -266,6 +266,18 @@ terminate a session
   The error code and string are optional; the default values are 0 and "".  The
   delivery of the error code and string MAY be best-effort.
 
+conclude a session
+: Indicate to the peer that a session no longer accepts new streams, but that
+  existing streams are allowed to complete.  The indication includes the
+  identifier of the last stream that is allowed to complete, enabling both
+  endpoints to agree on which stream's application data was processed.
```

Comment:
What does this correspond to?

Reply:
Did we have somewhere else tracking GOAWAY? 

We reference it throughout the document using actual HTTP/3 GOAWAY, so wanted to have a way to refer to that operation in transport-independent terms.

Reply:
Did we ever decide that HTTP/3 GOAWAY prevents you from opening new WT streams?  I do recall us having that discussion, not sure what we came up with.  I know for sure our implementation special-cases those and allows them through even if GOAWAY happened.